lineman/app/models/notification_records_interface.coffee

```diff
@@ -2,6 +2,13 @@ angular.module('loomioApp').factory 'NotificationRecordsInterface', (BaseRecords
   class NotificationRecordsInterface extends BaseRecordsInterface
     model: NotificationModel
 
+    fetchMyNotifications: ->
+      @fetch
+        params:
+          from: 0
```

Comment:
It might do to have a chat about paging with this system, which will be a bit wonky; I don't think we want to be combining timeframe and page too much. (Give me the 25th - 50th record created in the last day? Blerg. Even 'give me the last 20 records created in the last timeframe' doesn't work, because what if someone comes back after an hour and 30 things have been created? Still have a hole.)